src/partitioned_data/partitioned_data_pqn.jl

```diff
@@ -32,7 +32,16 @@ Gather the structures required to run a partitioned quasi-Newton trust-region me
 * `ps`: the partitioned step;
 * `pB`: the partitioned matrix (main memory cost);
 * `fx`: the current value of the objective function;
-* `name`: the name of partitioned quasi-Newton update peformed at each iterate.
+* `name`: the name of partitioned quasi-Newton update performed at each iteration:
+
+  * `name=:pbfgs`: every element-Hessian approximation is updated with BFGS;
+  * `name=:psr1`: every element-Hessian approximation is updated with SR1;
+  * `name=:pse`: every element-Hessian approximation is updated with BFGS or SR1 if the curvature condition doesn't hold;
```

Comment:
```suggestion
  * `name=:pse`: every element-Hessian approximation is updated with BFGS if the curvature condition holds, or with SR1 otherwise;
```